clarity/data/HOA_tools_cec2.py

```diff
@@ -12,9 +12,38 @@
 logger = logging.getLogger(__name__)
 
 
-# Code for generation ambisonic rotation matrices
+def convert_a_to_b_format(
+    flu: np.ndarray, frd: np.ndarray, bld: np.ndarray, bru: np.ndarray
+):
+    """Converts 1st order A format audio into 1st order B format
```

Comment:
In a similar vein does "1st order A format" and "1st order B format" have any more descriptive meaning that someone (e.g. me!) unfamiliar with what is actually being done might get a little more information from?